sched/adaptdl_sched/allocator.py

```diff
@@ -102,62 +184,66 @@ async def _find_nodes(self):
         node_template = NodeInfo(max_resources, True)
         return node_infos, node_template
 
+    def _get_job_info(self, job, job_infos, allocations=None):
+        if allocations and job.get("status", {}).get("phase") \
+                not in ["Pending", "Running", "Starting", "Stopping"]:
+            LOG.info("unkown status:", job, job.get("status", {}).get("phase"))
+            return
```

Comment:
Maybe pull this logic out, and only call `_get_job_info` if it's the right status.